topo_processor/cli/geostore/delete.py

```diff
@@ -0,0 +1,54 @@
+import boto3
+import click
+from linz_logger import LogLevel, get_log, set_level
+
+from topo_processor.geostore.invoke import invoke_lambda
+from topo_processor.util.time import time_in_ms
+
+
+@click.command()
+@click.option(
+    "-id",
+    "--datasetid",
+    required=True,
+    help="The dataset id to delete.",
+)
+@click.option(
+    "-p",
+    "--prod",
+    is_flag=True,
+    help="Use this flag to export into production environment.",
+)
+@click.option(
+    "-v",
+    "--verbose",
+    is_flag=True,
+    help="Use verbose to display trace logs",
+)
+def main(datasetid: str, prod: str, verbose: str) -> None:
```

Comment:
The different `geostore` wrapped function (`delete`, `export`, `list`, `status`) have some duplicate code. Any thoughts?

Reply:
Could it be one command e.g. `geostore-upload` and then a mode option i.e. `add`, `delete`, `list`, `status` followed by the specific flags for each mode e.g.  `--datasetid`.
Then some of the code could be reused as well.

Reply:
I had thought about that before but I did not want to have more arguments that there are already